hack/convert-yaml-to-env.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
```

Comment:
i dont get why we need it

Reply:
We don't, it's from one of the iterations, I'll remove it